src/internal/data-service-api/data-service-dialects-api.ts

```diff
@@ -131,6 +133,20 @@ export class DataServiceDialectsApiClient implements DataServiceDialectsApi {
         .then((it) => it.data),
     );
   }
+
+  findSummaryAll(query: FindDialectsSummaryDto): Promise<DialectsSummaryDto> {
+    return withReThrowingDataServiceError(
+      axios
+        .get<DialectsSummaryDto>(
+          `${this.baseUrl}/api/v1/dialects/summary/all`,
+          {
+            headers: createHeaders(),
+            ...(query && { params: query }),
```

Comment:
I think you always have query, so this can be simplified

Reply:
Good point, will adjust this too. Thanks!